docker-compose.yml

```diff
@@ -5,8 +5,8 @@ services:
     container_name: rabbitmq
     image: 'bitnami/rabbitmq:3.8.9'
     environment:
-      - RABBITMQ_USERNAME=${App__RabbitMQ__Username}
-      - RABBITMQ_PASSWORD=${App__RabbitMQ__Password}
+      - RABBITMQ_USERNAME=test
```

Comment:
This is purely for development docker-compose right?

Reply:
Correct. :)